src/main/java/frc/robot/subsystems/BallPath/Shooter/PIDShooterImpl.java

```diff
@@ -68,6 +97,72 @@ public void setShotPosition(ShotPosition shotPosition) {
         this.requestedPosition = shotPosition;
     }
 
+    public double getSetpointHood(double distance){
+        double hoodDif, distDif, difFromUpper, percentToAdd, amountToAdd;
+        double returnAmount = 0;
+        Integer Fender = 100_000;
```

Comment:
Replace `Integer` with simply `int`. There is no need for the boxed version here, it's just extra overhead for nothing.

Reply:
Ended up just commenting those out since they arent being used for anything